lib/util/src/arch.c

```diff
@@ -92,6 +92,10 @@ memlcpq(const void *s1, const void *s2, size_t len)
     return ((const char *)s1q - (const char *)s1);
 }
 
+#endif
+
+#if !__amd64__
+
```

Comment:
Heh!  I have that change in my sandbox :-)

Reply:
Preprocessor logic is so fun